### DISCUSSION TITLE
Publish action

### DISCUSSION THREAD

**Discussion by EMorf** [johndotpub/DiscordianAI#55] [General]
Seems like the Docker image was published, but I can't see it in the packages section:
![image](https://github.com/johndotpub/DiscordianAI/assets/8428597/94ca7a08-15c9-48b7-86f5-20196e850924)

Not sure if this is a repo setting, or if it is tied elsewhere. Is the token used as a secret tied to a different entity?

**Comment by johndotpub** [comment]
@EMorf Yeah it looks like some permission issues. I'll take a look.

**Comment by johndotpub** [comment]
This has been fixed. Please see issue #58 for the details.
